demo/demo_director.py

```diff
@@ -407,9 +407,12 @@ def register_vehicle_manifest_wrapper(
   be extracted before it is passed to the underlying director.py (in the
```

Comment:
Does the comment block starting on line 405 refer to the code just below it?  I don't see the XMLRPC binary object or how the raw data is extracted, or is it `signed_vehicle_manifest`?  And accessing signed_vehicle_manifest.data extracts the raw data? It might be clearer to refer to actual variables.